### PR TITLE
Improve Password-protected messages compliance

### LINKED ISSUE #2909: Password-protected messages compliance issue
I added `test-feature` string to array of disallowed strings, but Android app still allows to send password-protected messages with `test-feature` subject. Maybe regex generator splits string and expects `test` and `feature` as separate words?

Browser extension implementation handles this case correctly - showing error alert.
Currently `test-feature` is in the list of disallowed strings for `@flowcrypt.com` email accounts, so you can test it with your real FlowCrypt email.

### PR DESCRIPTION
**Pull request by DenBond7**
This PR Improved Password-protected messages compliance

close #2909 

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities

### CODE REVIEW

**Comment by DenBond7** [comment]
Should I make a new release?(1.6.1)

File: FlowCrypt/src/test/java/com/flowcrypt/email/api/retrofit/response/model/ClientConfigurationTest.kt [hunk 1]
```diff
@@ -18,11 +18,13 @@ class ClientConfigurationTest {
     val clientConfiguration = ClientConfiguration(
       disallowPasswordMessagesForTerms = listOf(
         "droid",
+        "test-feature",
         "[Classification: Data Control: Internal Data Control]",
       )
     )
 
     val regex = requireNotNull(clientConfiguration.getDisallowPasswordMessagesForTermsRegex())
+    println(regex)
```
Comment (sosnovsky) [inline FlowCrypt/src/test/java/com/flowcrypt/email/api/retrofit/response/model/ClientConfigurationTest.kt:27]: this line probably was needed only for debug and can be removed?

**Comment by sosnovsky** [comment]
> Should I make a new release?(1.6.1)

Yes, please create a new release and send updated build to customer, thanks!

File: FlowCrypt/src/test/java/com/flowcrypt/email/api/retrofit/response/model/ClientConfigurationTest.kt (hunk 1 above)
Comment (DenBond7) [reply FlowCrypt/src/test/java/com/flowcrypt/email/api/retrofit/response/model/ClientConfigurationTest.kt:27]: Yes, I will remove it via a separate PR.

**Pull request merged**